src/ServiceManager/Travely.ServiceManager.DAL/UnitOfWorks/UnitOfWork.cs

```diff
@@ -24,6 +26,17 @@ public IActivityRepository Activities
             }
         }
 
+        public IActivityTypeRepository ActivityTypes
```

Comment:
I would suggest naming it `ActivityTypeRepository` and to have standard `{ get; set; }`, then initialize it from ctor

Reply:
ok I will change the name, regarding to initialization in ctor, don't you think that will it be better to initialize the repository when we need it?

Reply:
I think that readability and maintainability are a priority comparing to that kind of optimization